doc/example-config

```diff
@@ -388,6 +388,18 @@
 #
 # feedhq-passwordfile "path-to-file"
 
+####  feedhq-passwordeval
+#
+# An even more secure alternative, is storing your password encrypted and
```

Comment:
This description is out of sync with the one in `configcommands.dsv`. The same goes for `oldreader-passwordeval` and `ttrss-passwordeval`.